specs/blocks.spec.ts

```diff
@@ -145,3 +145,31 @@ test('math is rendered in DropdownProblem question', async ({ page }) => {
   await page.locator('text=Check').click()
   await page.waitForSelector('#correct .MathJax')
 })
+
+test('math is rendered in MultipleChoiceProblem question', async ({ page }) => {
+  const htmlContent = `
+  <div class="os-raise-ib-pset" data-schema-version="1.0">
```

Comment:
I wasn't sure where to put this general comment, so shoving it here: As part of this PR let's make a couple of updates to `schema.md`:

1. Include details for the new question type
2. Specifically call out how to use MathJax math for `multiselect` and `multiplechoice` answer options + MathJax math rendering is not supported for `dropdown`

Reply:
Sounds good 👍 I'll append the new question type details, but I'd love to discuss the MathJax details from above at the engineering meeting too.

Reply:
@mjoss17 - Actually, it occurred to me just now that you can ignore my comment about adding `schema.md` changes to this PR. Moving forward let's spin up PRs to update the documentation **after** the code hits production. That way WW is not getting confused if they see / try things before the code ships.